pkg/store/store.go

```diff
@@ -127,6 +127,7 @@ type AnnouncementStore interface {
 
 // OrganizationStore describes how services interact with the Organization records.
 type OrganizationStore interface {
+	ListOrganizations() iterator.OrganizationIterator
```

Comment:
We will have to create a story for testing this -- I was only focused on the `bffutil` for now.

Reply:
story created